java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/gradle/GradleUtils.java

```diff
@@ -25,72 +18,28 @@ public final class GradleUtils {
     /**
      * Identifier for the plugin {@code com.github.sherter.google-java-format}.
      */
-    public static final String GOOGLE_JAVA_FORMAT_PLUGIN = Pattern.quote("com.github.sherter.google-java-format");
+    public static final String GOOGLE_JAVA_FORMAT_PLUGIN = "com.github.sherter.googlejavaformatgradleplugin.GoogleJavaFormatPlugin";
```

Comment:
I know it was not added in this PR, but why does the presence of this plugin trigger JDK11? Does this require JDK 11 as a baseline?

Reply:
It doesn't work on Java 17, at least out of the box, so it's just meant to try Java 11 first.

It should be possible to support it on Java 17 with JVM args with the right `--add-exports` flags.